src/stars.css

```diff
@@ -0,0 +1,35 @@
+.card__reviews {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-top: 16px;
+  padding-left: 16px;
+  padding-right: 16px;
+  order: 2;
+}
+
+.reviews__info {
+  font-weight: 400;
+  font-size: 10px;
+  color: #060b35;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(./images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  height: 16px;
+  width: 16px;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--4 :nth-child(-n+4) {
+  background: url(./images/star-active.svg) no-repeat center;
+}
```

Comment:
why only for `stars--4`? it's need to be for all counts of stars (5)

Reply:
didn't change it

Reply:
didn't change it